new rule: may use asynchronous processing (#787)
zalando/restful-api-guidelines: chapters/http-status-codes-and-errors.adoc

==== 202 Accepted {rfc-status-202} {use} {document} {POST} {PUT} {PATCH} {DELETE}
[.indent]
The request was successful and will be processed asynchronously.

> Only applicable to methods which change something, with the exception of 
> GET methods provided for resources still being asynchronously processed,
> as not recommended in <<253>>.

Reviewer: ```suggestion
Only applicable to methods which change something, with the exception of 
{GET} methods indicating that a resources is still created asynchronously
as described in <<253>>.
```
